cohortextractor/patients.py

```diff
@@ -2165,6 +2165,7 @@ def outpatient_appointment_date(
             * `binary_flag`: indicates if they have had an outpatient appointment or not
             * `date`: earliest date of outpatient appointment within the specified period
             * `number_of_matches_in_period`: number of outpatient appointments in period
+            * `consultation_medium_used`: consultation medium code for the earliest outpatient appointment within the specified period
```

Comment:
Could we point to where the possible codes are documented?

Reply:
asked at https://github.com/opensafely-core/cohort-extractor/issues/492#issuecomment-910365078

Reply:
added a link